src/pytest_postgresql/__init__.py

```diff
@@ -19,8 +19,17 @@
 """Main module for pytest-postgresql."""
 
 import logging
+import sys
 
 __version__ = '1.2.0'
 
 
 logger = logging.getLogger(__name__)
+
+try:
+    if sys.implementation.name == "pypy":
+        import psycopg2cffi.compat
+        psycopg2cffi.compat.register()
+except AttributeError:
```

Comment:
Can we use `platform.python_implementation()` instead of `sys.implementation.name`? It works on 2.7.

I don't like catching exceptions around imports, problems in `psycopg2cffi` itself should not be hidden from our users.

Do we need compat, or could we conditionally import any available of psycopg2 or psycopg2cffi?